presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/Launcher.java

```diff
@@ -155,7 +155,10 @@ protected Extensions getExtensions()
         @Override
         protected Object[][] getContents()
         {
-            return new Object[][]{{"project.version", readProjectVersion()}};
+            return new Object[][]{
+                    {"project.version", readProjectVersion()},
+                    {"product.tests.module", "presto-product-tests"},
```

Comment:
Let's call this `product-tests.module` since we use dots for hierarchy, but "product tests" is a single name and not a hierarchy.